storage/simulation/cluster.go

```diff
@@ -64,6 +69,8 @@ func createCluster(stopper *stop.Stopper, nodeCount int) *Cluster {
 		nodes:         make(map[proto.NodeID]*Node),
 		stores:        make(map[proto.StoreID]*Store),
 		ranges:        make(map[proto.RangeID]*Range),
+		rand:          rand,
+		seed:          seed,
```

Comment:
I'm just storing it for now, but the plan is to output as a header to the output file and accept a seed flag so we can repeat simulation runs later on.